Makefile

```diff
@@ -29,9 +29,9 @@ lint:
 
 test:
 ifeq ($(BUILD_IN_CONTAINER),1)
-	$(DOCKER_RUN) $(GOLANG_CONTAINER) go test ./...
+	$(DOCKER_RUN) $(GOLANG_CONTAINER) go test -race ./...
```

Comment:
this change and the one in line 34 have nothing to do with the PR. please remove.